### ISSUE TITLE
why  there is only two softmax layer in prototxt

### ISSUE THREAD

**Issue by ZhiweiYan-96** [ustc-vim/dualnet#6]
hi~

I have read your paper, but  I only find two softmaxloss layer is ' res_e1_train_val.prototxt' . But there are  three softmaxloss Layers in paper.  May be it is due to 'iterative training'  ?  Thanks for your help.

**Comment by hshustc** [comment]
In the iterative training, there are two losses involved. And in most cases, it is enough to set max_iter=1 to obtain the perform gain. Besides, the way to train with max_iter=2 is similar.

**Comment by ZhiweiYan-96** [comment]
So if the max_iter=2.   I need three prototxt files which one of these is  used in ite_0 and the other two are used in  ite_2k and ite2kpulus1 ?

**Comment by ZhiweiYan-96** [comment]
The other problem is when three losslayer are needed? The visualization of DualNet has three losslayers in supplementary material.  thxs

**Comment by hshustc** [comment]
1. In each training phase, only one prototxt is needed in my implementation.
2. Three loss layers are need in the jointly finetuning. The loss function is presented in the paper.

**Comment by ZhiweiYan-96** [comment]
thanks for your response!  i will read it again.
